easybuild/easyconfigs/p/PRINSEQ/PRINSEQ-0.20.4-goolf-1.4.10-Perl-5.16.3.eb

```diff
@@ -0,0 +1,111 @@
+         'source_urls': ['https://cpan.metacpan.org/authors/id/X/XA/XAOC/'],
+    }),
+    ('ExtUtils::PkgConfig', '1.15', {
+         'source_tmpl': 'ExtUtils-PkgConfig-1.15.tar.gz',
+         'source_urls': ['https://cpan.metacpan.org/authors/id/X/XA/XAOC/'],
+    }),
+    ('Getopt::Long', '2.48', {
+         'source_tmpl': 'Getopt-Long-2.48.tar.gz',
+         'source_urls': ['https://cpan.metacpan.org/authors/id/J/JV/JV/'],
+    }),
+    ('Pod::Usage', '1.68', {
+        'source_tmpl': 'Pod-Usage-1.68.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MAREKR/'],
+    }),
+    ('File::Temp', '0.2304', {
+        'source_tmpl': 'File-Temp-0.2304.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/'],
+    }),
+    ('Digest::MD5', '2.54', {
+        'source_tmpl': 'Digest-MD5-2.54.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/G/GA/GAAS/'],
+    }),
+    ('File::Spec', '3.62', {
+        'source_tmpl': 'PathTools-3.62.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/R/RJ/RJBS/'],
+    }),
+    ('JSON', '2.90', {
+        'source_tmpl': 'JSON-2.90.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MAKAMAKA/'],
+    }),
+    ('Cairo', '1.106', {
+        'source_tmpl': 'Cairo-1.106.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XA/XAOC/'],
+    }),
+    ('Statistics::PCA', '0.0.1', {
+        'source_tmpl': 'Statistics-PCA-0.0.1.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DS/DSTH/'],
+    }),
+    ('MIME::Base64', '3.15', {
+        'source_tmpl': 'MIME-Base64-3.15.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/G/GA/GAAS/'],
+    }),
+    ('Math::Cephes::Matrix', '0.5304', {
+        'source_tmpl': 'Math-Cephes-0.5304.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/S/SH/SHLOMIF/'],
+    }),
+    ('Math::MatrixReal', '2.12', {
+        'source_tmpl': 'Math-MatrixReal-2.12.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/L/LE/LETO/'],
+    }),
+    ('Text::SimpleTable', '2.03', {
+        'source_tmpl': 'Text-SimpleTable-2.03.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MR/MRAMBERG/'],
+    }),
+    ('Contextual::Return', '0.004008', {
+        'source_tmpl': 'Contextual-Return-0.004008.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DC/DCONWAY/'],
+    }),
+    ('Want', '0.26', {
+        'source_tmpl': 'Want-0.26.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/R/RO/ROBIN/'],
+    }),
+]
+
+modextrapaths = {
+    'PATH': "", # add installation dir to PATH
+    'PERL5LIB': 'lib/perl5/site_perl/%s/' % (perlver)
+}
+
+postinstallcmds = ["sed -i -e 's|/usr/bin/perl|/usr/bin/env\ perl|' %(installdir)s/*.pl", # fix shebang line
+                   "chmod +x %(installdir)s/*.pl"]  # add execution permission
```

Comment:
This actually needs to be moved to the generic Perl extension block. The Python one implements patching wrong shebangs.